### PR TITLE
Bitfinex: Ensure websocket configuration is sent with every connection

### PR DESCRIPTION
**Pull request by Beadko**
# PR Description

Fix WebSocket configured more than once

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [x] go test ./... -race
- [x] golangci-lint run
- [ ] Test X

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/bitfinex/bitfinex_websocket.go [hunk 1]
```diff
@@ -1540,13 +1538,6 @@ func (b *Bitfinex) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription,
 
 // Subscribe sends a websocket message to receive data from the channel
 func (b *Bitfinex) Subscribe(channelsToSubscribe []stream.ChannelSubscription) error {
-	checksum := make(map[string]interface{})
-	checksum["event"] = "conf"
-	checksum["flags"] = bitfinexChecksumFlag + bitfinexWsSequenceFlag
-	err := b.Websocket.Conn.SendJSONMessage(checksum)
-	if err != nil {
-		return err
-	}
```
Comment (shazbert) [inline exchanges/bitfinex/bitfinex_websocket.go:1541]: rm whitespace as the linter is complaining.

File: exchanges/bitfinex/bitfinex_websocket.go [hunk 2]
```diff
@@ -1568,6 +1559,15 @@ func (b *Bitfinex) Subscribe(channelsToSubscribe []stream.ChannelSubscription) e
 	return errs
 }
 
+// ConfigureWS to send checksums and sequence numbers
+func (b *Bitfinex) ConfigureWS() error {
+	checksum := make(map[string]interface{})
+	checksum["event"] = "conf"
+	checksum["flags"] = bitfinexChecksumFlag + bitfinexWsSequenceFlag
+
+	return b.Websocket.Conn.SendJSONMessage(checksum)
```
Comment (shazbert) [inline exchanges/bitfinex/bitfinex_websocket.go:1568]: Suggestion: Could even consolidate this even more if you so choose but I will leave that up to you.
```go
	return b.Websocket.Conn.SendJSONMessage(map[string]interface{}{
		"event": "conf",
		"flags": bitfinexChecksumFlag + bitfinexWsSequenceFlag,
	})
```

**Review by shazbert — CHANGES_REQUESTED** [review]
Nice work this is pretty much tACK with one linter issue.

File: exchanges/bitfinex/bitfinex_websocket.go (hunk 1 above)
Comment (Beadko) [reply exchanges/bitfinex/bitfinex_websocket.go]: Thanks for that, i ran a linter test before and had no issues for some reason. All done

File: exchanges/bitfinex/bitfinex_websocket.go (hunk 2 above)
Comment (Beadko) [reply exchanges/bitfinex/bitfinex_websocket.go]: done

**Review by shazbert — APPROVED** [review]
Thanks Bea! 👍

**Review by gloriousCode — APPROVED** [review]
Makes sense, tACK!

**Review by thrasher- — APPROVED** [review]
Thanks @Beadko !

**Pull request merged**